docs/release_notes/v0.9.0.md

```diff
@@ -0,0 +1,396 @@
+  
+# Dapr 0.9.0
+
+We're happy to announce the release of Dapr 0.9.0!
+
+**Highlights**
```

Comment:
This section seems to be from previous release. Can we update this.

Reply:
+1

Reply:
fixed